### ISSUE TITLE
Unexpected child with tag 'OME:Plane'

### ISSUE THREAD

**Issue by schienstockd** [tlambert03/ome-types#135]
Hello,

I created an OME-ZARR from a CZI file with bioformats2raw with 2 pyramid scales.
Then, I am trying to read the METADATA.ome.xml with `from_xml` and I get the following error:

```
XMLSchemaChildrenValidationError: failed validating <Element '{http://www.openmicroscopy.org/Schemas/OME/2016-06}Pixels' at 0x7fc7b75d4720> with XsdGroup(model='sequence', occurs=[1, 1]):

Reason: Unexpected child with tag 'OME:Plane' at position 5. Tag ('OME:BinData' | 'OME:TiffData' | 'OME:MetadataOnly') expected.

Schema:

  <xsd:complexType xmlns:xsd="http://www.w3.org/2001/XMLSchema">
    <xsd:sequence>
      <xsd:element ref="Channel" minOccurs="0" maxOccurs="unbounded">
        <xsd:annotation>
          <xsd:appinfo>
            <xsdfu>
              <ordered />
            </xsdfu>
          </xsd:appinfo>
        </xsd:annotation>
      </xsd:element>
      <xsd:choice minOccurs="1" maxOccurs="1">
        <xsd:element ref="BinData" minOccurs="1" maxOccurs="unbounded" />
        <xsd:element ref="TiffData" minOccurs="1" maxOccurs="unbounded" />
        <xsd:element ref="MetadataOnly" minOccurs="1" maxOccurs="1" />
      </xsd:choice>
      <xsd:element ref="Plane" minOccurs="0" maxOccurs="unbounded" />
    </xsd:sequence>
    <xsd:attribute name="ID" use="required" type="PixelsID" />
    <xsd:attribute name="DimensionOrder" use="required">
    ...
    ...
  </xsd:complexType>

Instance:

  <Pixels xmlns="http://www.openmicroscopy.org/Schemas/OME/2016-06" BigEndian="true" DimensionOrder="XYZCT" ID="Pixels:0" Interleaved="false" PhysicalSizeX="0.35143526946023074" PhysicalSizeXUnit="µm" PhysicalSizeY="0.35143526946023074" PhysicalSizeYUnit="µm" PhysicalSizeZ="2.0103026803688864" PhysicalSizeZUnit="µm" SignificantBits="16" SizeC="4" SizeT="1" SizeX="9536" SizeY="6944" SizeZ="1" Type="uint16"><Channel AcquisitionMode="LaserScanningConfocalMicroscopy" Color="16711935" EmissionWavelength="526.2136399999998" EmissionWavelengthUnit="nm" ExcitationWavelength="488.00000000000006" ExcitationWavelengthUnit="nm" Fluor="FITC" ID="Channel:0:0" IlluminationType="Epifluorescence" Name="ChS1-T1" SamplesPerPixel="1"><DetectorSettings Binning="1x1" ID="Detector:0:0" /><LightPath /></Channel><Channel AcquisitionMode="LaserScanningConfocalMicroscopy" Color="-1" EmissionWavelength="690.5" EmissionWavelengthUnit="nm" ExcitationWavelength="633.0" ExcitationWavelengthUnit="nm" Fluor="APC" ID="Channel:0:1" IlluminationType="Epifluorescence" Name="Ch2-T1" SamplesPerPixel="1"><DetectorSettings Binning="1x1" ID="Detector:0:1" /><LightPath /></Channel><Channel AcquisitionMode="LaserScanningConfocalMicroscopy" Color="65535" EmissionWavelength="448.38500000000005" EmissionWavelengthUnit="nm" ExcitationWavelength="405.00000000000006" ExcitationWavelengthUnit="nm" Fluor="Pacific Blue" ID="Channel:0:2" IlluminationType="Epifluorescence" Name="Ch1-T2" SamplesPerPixel="1"><DetectorSettings Binning="1x1" ID="Detector:1:0" /><LightPath /></Channel><Channel AcquisitionMode="LaserScanningConfocalMicroscopy" Color="-16776961" EmissionWavelength="594.452559" EmissionWavelengthUnit="nm" ExcitationWavelength="561.0" ExcitationWavelengthUnit="nm" Fluor="R-PE" ID="Channel:0:3" IlluminationType="Epifluorescence" Name="ChS2-T2" SamplesPerPixel="1"><DetectorSettings Binning="1x1" ID="Detector:1:1" /><LightPath /></Channel><Plane DeltaT="16416.086416242422" DeltaTUnit="s" PositionX="16628.8" PositionXUnit="µm" PositionY="-787.25" PositionYUnit="µm" PositionZ="34.93" PositionZUnit="µm" TheC="0" TheT="0" TheZ="0" /><Plane DeltaT="16416.086416242422" DeltaTUnit="s" PositionX="16628.8" PositionXUnit="µm" PositionY="-787.25" PositionYUnit="µm" PositionZ="34.93" PositionZUnit="µm" TheC="1" TheT="0" TheZ="0" /><Plane DeltaT="16416.086416242422" DeltaTUnit="s" PositionX="16628.8" PositionXUnit="µm" PositionY="-787.25" PositionYUnit="µm" PositionZ="34.93" PositionZUnit="µm" TheC="2" TheT="0" TheZ="0" /><Plane DeltaT="16416.086416242422" DeltaTUnit="s" PositionX="16628.8" PositionXUnit="µm" PositionY="-787.25" PositionYUnit="µm" PositionZ="34.93" PositionZUnit="µm" TheC="3" TheT="0" TheZ="0" /></Pixels>
[METADATA.ome.xml.txt](https://github.com/tlambert03/ome-types/files/8329415/METADATA.ome.xml.txt)

Path: /OME/Image/Pixels
```

Here the whole XML:
[METADATA.ome.xml.txt](https://github.com/tlambert03/ome-types/files/8329418/METADATA.ome.xml.txt)

Can I skip the validation somehow? I have version `0.2.10`.

**Comment by tlambert03** [comment]
Hi @schienstockd,

The option to skip validation was recently added in #127, so the current main branch actually _does_ work with this file:

```python
In [18]: o = from_xml('/Users/talley/Downloads/METADATA.ome.xml')

In [19]: o
Out[19]:
OME(
   experimenters=[<1 Experimenters>],
   images=[<1 Images>],
   instruments=[<1 Instruments>],
   structured_annotations=[<240 Structured_Annotations>],
)

In [20]: o.images
Out[20]:
[
    Image(
   id='Image:0',
   name='',
   pixels=Pixels(
      id='Pixels:0',
      dimension_order='XYZCT',
      size_c=4,
      size_t=1,
      size_x=9536,
      size_y=6944,
      size_z=1,
      type='uint16',
      big_endian=True,
      channels=[<4 Channels>],
      interleaved=False,
      physical_size_x=0.35143526946023074,
      physical_size_y=0.35143526946023074,
      physical_size_z=2.0103026803688864,
      planes=[<4 Planes>],
      significant_bits=16,
   ),
   experimenter_ref=ExperimenterRef(id='Experimenter:0'),
   instrument_ref=InstrumentRef(id='Instrument:0'),
   objective_settings=ObjectiveSettings(
      id='Objective:0',
      medium='Oil',
      refractive_index=1.518,
   ),
   stage_label=StageLabel(
      name='Scene position #0',
      x=16628.8,
      x_unit='µm',
      y=-787.25,
      y_unit='µm',
      z=34.93,
      z_unit='µm',
   ),
)
]
```

It only works for the new lxml parser (which is now the default parser).  Note also, writing the resulting OME object out to xml  with `to_xml` is still failing.  

Not sure how soon we'll release the new version with the changes in the main branch, since it is a potentially breaking change in the behavior of the library.  So it needs to be given some thought.

**Comment by tlambert03** [comment]
I'm going to close this, since it has a solution already merged, and the writing can be discussed in #129. 
@schienstockd, note that after #138 merges (before the upcoming release), you'll need to explicitly opt in to the new parser with  `o = from_xml('METADATA.ome.xml', parser='lxml')`.

Feel free to comment/reopen if I've missed something that we still need to fix for your case!
